domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidInntektsmelding/dto/InntektsmeldingDto.java

```diff
@@ -31,6 +31,5 @@ public record InntektsmeldingDto(BigDecimal inntektPrMnd,
                                  List<NaturalYtelse> bortfalteNaturalytelser,
                                  List<Refusjon> refusjonsperioder,
                                  InntektsmeldingInnsendingsårsak innsendingsårsak,
-                                 List<UUID> behandlingsIdeer,
```

Comment:
Fjerner slik at det kun finnes 1 liste som vil være  hvilke behandlinger en IM er aktive for. Kanskje misvisende å kalle det "aktiveBehandlingsIder". Forslag?

Reply:
tilknyttedeBehandlinger?

Reply:
Bra forslag